packages/cactus-plugin-ledger-connector-quorum/src/main/json/openapi.json

```diff
@@ -434,6 +434,19 @@
                         "default": 60000,
                         "nullable": false
                     },
+                    "constructorArgs": {
+                        "type": "object",
+                        "example": "{} - An empty object literal can be sufficient if your contract does not have parameters.",
+                        "nullable": false,
+                        "properties": {
+                            "Args": {
```

Comment:
@travis-payne Please use `camelCase` for variable names.

Other question: Is there a reason why we are wrapping the array of arguments in an object? IIRC the array itself would be enough, but I'm not a 100% sure either way so just asking this one.

Reply:
I believe I was following the pattern that existed in supply-chain-app-dummy-infrastructure.ts

https://github.com/hyperledger/cactus/blob/1a5edeae834bc275252e588379f214324977a3ff/examples/cactus-example-supply-chain-backend/src/main/typescript/infrastructure/supply-chain-app-dummy-infrastructure.ts#L289

I did think it a little odd (both the camelCase and wrapping in an object), but presumed there may of been a reason for this, I can make the changes across both packages?

Reply:
Fixed, haven't made changes in other packages as that should probably be another PR